packages/components/src/components/form/Select/index.tsx

```diff
@@ -441,8 +436,6 @@ export const Select = ({
             <ReactSelect
                 ref={selectRef}
                 onKeyDown={onKeyDown}
-                onMenuOpen={() => setIsMenuOpen(true)}
```

Comment:
@dahaca do you remember, what was the reason to add the `isMenuOpen` state?

Reply:
@dahaca clarification would be nice, because I see no difference in UI

Reply:
Sorry, I missed the notifications :( This is related to closing Select on scroll. The event listener is not added if Select is not open.

Reply:
Ah okay, so I removed the logic and replaced it by `closeMenuOnScroll` prop